### ISSUE TITLE
Publish to npm?

### ISSUE THREAD

**Issue by percyhanna** [ggozad/Backbone.cachingSync#17]
From what I could tell, this package doesn't appear to be published to `npm`. Also, Backbone and Underscore support AMD natively now.

It'd be nice to support `npm` officially. As well, using the `jquery` dependency instead of `jquery-browser` should work, I think.

**Comment by ggozad** [comment]
Absolutely. I am away on holidays, it would be awesome if you could make a PR with the updates necessary.
